README.md

```diff
@@ -192,6 +192,34 @@ Example config:
 
 To import a saved files, use XHGui's provided `external/import.php` script.
 
+### Stack saver
```

Comment:
I've seen other libraries and tools call this an 'aggregate'. Would that term be applicable here?

Reply:
I can rename to aggregate. can you point to examples, so I'll update PR body as well.